package.json

```diff
@@ -17,13 +17,13 @@
   ],
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha ./tests/lib/**/*.js",
     "new-rule-format": "node ./bin/eslint-transforms.js -t ./lib/new-rule-format/new-rule-format.js",
```

Comment:
Any ideas on what this script is doing? Per the [new standard](https://eslint.org/docs/latest/contribute/package-json-conventions#main-script-names), with the exception of npm life cycle scripts all scripts should begin with `build:`, `release:`, `lint:`, `start:`, or `test:`.

Reply:
I think we can prefix it with `build:`, it seems to be [updating rule from the old format to new format](https://github.com/eslint/eslint-transforms/blob/b044f93a74d7f1193878f2fadbaf57a3509761d6/lib/new-rule-format/new-rule-format.js#L2)

Reply:
Hmm, it doesn't seem to fit any of the `build:`, `release:`, `lint:`, `start:`, or `test:` categories. Should we keep it as-is?

Reply:
I think we can just remove it.

Reply:
> I think we can just remove it.

Agreed.

Reply:
removed.